src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,382 @@
+---
+title: EKS Fargate (beta)
+watermark: BETA
+---
+
+In order to monitor Kubernetes workloads scheduled on Fargate, our solution automatically injects a sidecar containing the infrastructure agent and the `nri-kubernetes` integration in each pod that needs to be monitored.
+
+If the same Kubernetes cluster also contains EC2 nodes, our solution is deployed as a `DaemonSet` in all of them. No sidecar will be injected into pods scheduled in EC2 nodes, and no `DaemonSet` will be deployed to Fargate nodes. Here's an example of a hybrid instance with both Fargate and EC2 nodes:
+
+![Diagram showing an EKS cluster with Fargat and EC2 nodes](./images/fargate-overview.svg "Diagram showing an EKS cluster with Fargat and EC2 nodes")
+<figcaption>
+In a mixed environment, the integration only uses a sidecar for Fargate nodes.
+</figcaption>
+
+New Relic collects all the supported metrics for all Kubernetes objects regardless of where they are scheduled, whether it's Fargate or EC2 nodes. Please note that, due to Fargate limitations, the New Relic integration is limited to running in [unprivileged](/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure/#unprivileged) mode on Fargate nodes. This means that metrics that are usually fetched from the host directly, like running processes, will not be available for Fargate nodes.
+
+The agent in both scenarios will scrape data from Kube State Metrics (KSM), Kubelet, and cAdvisor and send data in the same format.
+
+<Callout variant="important">
+You are still required to deploy and monitor a Kube State Metrics instance, which the Helm Chart can do automatically if instructed to. KSM will be monitored as any other workload: By injecting a sidecar if it gets scheduled in a Fargate node or with the local instance of the `DaemonSet` if it gets scheduled on an EC2 node.
+</Callout>
+
+Other components such as `nri-prometheus`, `nri-metadata-injection`, and `nri-kube-events` do not need any extra steps to be deployed on Fargate.
+
+<Callout variant="tip">
+New Relic Logging is not yet supported on Fargate nodes.
+</Callout>
+
+## Installation [#installation]
+
+You can choose between two alternatives for installing New Relic full observability in your EKS Fargate cluster: 
+
+* [Automatic (recommended)](#automatic)
+* [Manual injection](#manual)
+
+Regardless of the approach you choose, the experience is exactly the same after it's installed. The only difference is how the container is injected.
```

Comment:
```suggestion
Regardless of the approach you choose, the experience is exactly the same after it's installed. The only difference is how the container is injected. We do recommend setting up automatic injection with the New Relic Infrastructure Operator since it will eliminate the need to manually edit each deployment you want to monitor.
```

Reply:
I think it's important to brag about our fantastic operator, which at the moment is a competitive advantage w.r.t. other solutions in the market.

Reply:
@roobre, great suggestion!